src/System.Diagnostics.EventLog/src/Configurations.props

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netcoreapp-Windows_NT;
+      netcoreapp2.0-Windows_NT;
```

Comment:
@weshaggard won't this break the netcoreapp build where we don't build out the entire RefPath folder?  Typically we ensure that all projects that support some version of netcoreapp have a version-less configuration for the vertical-version-less build leg.

Reply:
Your probably right. I wonder how many we have in this case. I suspect a number of the compat libraries are there. This will likely break our source-build so I will file an issue for that.